lib/instrumentation/memcached.js

```diff
@@ -69,12 +70,12 @@ module.exports = function initialize(agent, memcached, moduleName, shim) {
     }
 
     // finally, execute the original command
-    return {
+    return new OperationSpec({
       name: metacall.type || 'Unknown',
       callback: function wrapCallback(shim, fn, fnName, opSegment) {
         shim.bindCallbackSegment(metacall, 'callback', opSegment)
       },
-      parameters: parameters
-    }
+      parameters
```

Comment:
FYI, once my PR is merged, we will have https://github.com/jsumners-nr/node-newrelic/blob/spec-migration/lib/shim/specs/params/datastore.js